web/settings.py

```diff
@@ -143,7 +145,14 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
 STATIC_URL = "/static/"
-STATIC_ROOT = env("STATIC_ROOT", default="/web/staticfiles")
+STATIC_ROOT = BASE_DIR / "staticfiles"
```

Comment:
This is something my PR takes care of https://github.com/Comp-490-SeniorProject/site/pull/15/commits/70280709c48dc92e19e704a044c293c7dddc7dd3 though it isn't merged yet.

Reply:
Oh ok I'll make the adjustment.